backend/variants/views/api/export.py

```
@@ -27,6 +30,50 @@
+        # Use LibreOffice to convert DOCX to PDF
+        cmd = [
+            "libreoffice",
+            "--headless",
+            "--convert-to",
+            "pdf",
+            "--outdir",
+            os.path.dirname(pdf_path),
+            docx_path,
+        ]
+
+        result = subprocess.run(cmd, capture_output=True, text=True, timeout=30)
```

Ruff (0.13.1)

`subprocess` call: check for execution of untrusted input

(S603)